thirdeye-ui/src/app/routers/general-unauthenticated/general-unauthenticated.router.tsx

```diff
@@ -23,20 +23,12 @@ const LoginPage = lazy(() =>
     ).then((module) => ({ default: module.LoginPage }))
 );
 
-const SwaggerDocs = lazy(() =>
-    import(/* webpackChunkName: "swagger-page" */ "../../pages/swagger").then(
```

Comment:
Is this page being used anywhere else? If not, can we remove it?

Reply:
Swagger page?